bitbots_vision/src/bitbots_vision/vision_modules/candidate.py

```diff
@@ -145,6 +152,17 @@ def point_in_candidate(self, point):
                 <= point[1]
                 <= self.get_upper_left_y() + self.get_height())
 
+    @staticmethod
+    def sort_candidates(candidatelist):
```

Comment:
It would be better to make clear in the name of the method that it sorts by rating.